web/components/sections/ReadyToInvest/ReadyToInvest.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-debugger, no-console */
+import React from 'react'
+import PropTypes from 'prop-types'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import { Grid, Container, Box, CssBaseline } from '@mui/material'
+import SimpleBlockContent from '../../SimpleBlockContent'
+import styles from './ReadyToInvest.module.css'
+import RedirectButton from '../../RedirectButton/RedirectButton'
+
+
+const theme = createTheme()
+
+function ReadyToInvest(props) {
+  const { heading, buttonText, backgroundColor } = props
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Box sx={{
+        backgroundColor: backgroundColor && backgroundColor
+      }}>
+        <Container sx={{ maxWidth: { sm: 'md', md: 'md', lg: 'lg' } }} >
+          <Grid container pt={6} pb={8}>
+            <CssBaseline />
+            {
+              heading && (
+                <Grid item xs={12} mb={4}>
+                  <div className={styles.simpleBlockContent}>
+                    <SimpleBlockContent blocks={heading} />
+                  </div>
+                </Grid>
+              )
+            }
+            <Grid item xs={12} mx={'auto'} display={'inline-flex'} justifyContent={'center'}>
+              <RedirectButton
+                title={<SimpleBlockContent blocks={buttonText} />}
```

Comment:
You're using a button with only the text. The button won't redirect anywhere when clicked. Check an example in `web\components\sections\HeroDoubleButton\HeroDoubleButton.js`